person.rb

```diff
@@ -2,7 +2,7 @@
 
 # Person Class
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ] Getter and setter for `:rentals` should be present, not just getter. Kindly refactor accordingly.

Reply:
This issue is well resolved! ✔️